cos-fleetshard-operator-debezium/pom.xml

```diff
@@ -119,6 +119,13 @@
             <plugin>
                 <groupId>io.quarkus</groupId>
                 <artifactId>quarkus-maven-plugin</artifactId>
+                <dependencies>
+                    <dependency>
+                        <groupId>org.bf2</groupId>
+                        <artifactId>cos-fleetshard-api-deployment</artifactId>
+                        <version>${cos-fleetshard-api.version}</version>
+                    </dependency>
+                </dependencies>
```

Comment:
Like this! As plugin dependency! @lburgazzoli

Reply:
can you do the same on the camel module ?

Reply:
Done